src/utils/simplified-account/validation/webhook.schema.js

```diff
@@ -0,0 +1,40 @@
+const { body } = require('express-validator')
+
+const WEBHOOK_DESCRIPTION_MAX_LENGTH = 40
```

Comment:
Even though we have this, it would be good to limit the text box in the njk file to 40 chars too. That might save a round trip for the user.